### PR TITLE
Enhancements to CSP page

### PR DESCRIPTION
**Pull request by shaialon**
- Violation reporting.
- CSP scanning.

### CODE REVIEW

File: files/en-us/web/http/csp/index.html [hunk 1]
```diff
@@ -212,15 +212,15 @@ <h2 id="Violation_report_syntax">Violation report syntax</h2>
   <dt><code>document-uri</code></dt>
   <dd>The URI of the document in which the violation occurred.</dd>
   <dt><code>effective-directive</code></dt>
-  <dd>The directive whose enforcement caused the violation.</dd>
+  <dd>The directive whose enforcement caused the violation. Some browsers may provide different values, such as Chrome providing <code>style-src-elem</code>/<code>style-src-attr</code>, even when the actual enforced directive was <code>style-src</code></dd>
```
Comment (Elchi3) [inline files/en-us/web/http/csp/index.html:215]:
```suggestion
  <dd>The directive whose enforcement caused the violation. Some browsers may provide different values, such as Chrome providing <code>style-src-elem</code>/<code>style-src-attr</code>, even when the actual enforced directive was <code>style-src</code>.</dd>
```

File: files/en-us/web/http/csp/index.html [hunk 2]
```diff
@@ -305,4 +305,6 @@ <h2 id="See_also">See also</h2>
   </li>
   <li><a href="https://github.com/google/csp-evaluator">CSP Evaluator</a> - Evaluate your
     Content Security Policy</li>
+  <li><a href="https://cspscanner.com/">CSP Scanner</a> - Improve your
```
Comment (Elchi3) [inline files/en-us/web/http/csp/index.html:308]: On this one I like @chrisdavidmills to comment. I was looking for our editorial guidelines about linking to commercial sites and I think they are now here: https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Editorial

Chris, what do you think?

**Review by Elchi3 — CHANGES_REQUESTED** [review]
Welcome to mdn/content and thanks for your contribution! I've made some suggestions but LGTM otherwise.
Bonus points if you follow up and also update https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy-Report-Only with this information.

Also, I think CSP report-sample should be documented as it looks missing to me.

File: files/en-us/web/http/csp/index.html (hunk 2 above)
Comment (shaialon) [reply files/en-us/web/http/csp/index.html:308]: Thanks @Elchi3 . Just for emphasis:
**CSP scanner** is a free utility for educational purposes (as site/ extension) and can be used by anyone - much like the Google CSP evaluator above, with some extra functionality.

CSP scanner is sponsored by [RapidSec](https://rapidsec.com/), which does have commercial intent ,  yet the scanner remains open to anyone - including RapidSec competitors' users.

I believe that this is standard practice in such cases.

Comment (Ryuno-Ki) [reply files/en-us/web/http/csp/index.html:308]: Could you add a `class="external"` to external links? That would help us add some design elements in the future :-)

Comment (shaialon) [reply files/en-us/web/http/csp/index.html]: @Ryuno-Ki Sure thing, added 👍

Comment (Ryuno-Ki) [reply files/en-us/web/http/csp/index.html]: Thanks!

**Comment by shaialon** [comment]
@Elchi3 I think we are all good. Submitted for re-review.

**Comment by Elchi3** [comment]
Thanks for your quick follow-up, @shaialon 👍 
I appreciate you elaborating on the link addition and explaining what your service does 🚀 

I still like to hear from Chris about our editorial guideline, so that I know for my future reviews.

**Comment by chrisdavidmills** [comment]
> I still like to hear from Chris about our editorial guideline, so that I know for my future reviews.

I think this one is OK — it is a free tool, and actually useful and relevant.

**Review by chrisdavidmills — APPROVED** [review]

**Review by Elchi3 — APPROVED** [review]
okidoki! Thanks Chris! 👍

**Pull request merged**